Bookings/Booking.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Airport_Ticket_Booking_System.Flights;
+using Airport_Ticket_Booking_System.Users;
+
+namespace Airport_Ticket_Booking_System.Bookings;
+
+public class Booking
+{
+    public int Id { get; set; }
+    public Flight Flight { get; set; }
+    public User User { get; set; }
+    public DateTime BookingDate { get; set; } = DateTime.Now;
+    public BookingStatus Status { get; set; } = BookingStatus.Pending;
+
+    public static string ToCsv(Booking booking)
+    {
+        return $"{booking.Id},{booking.Flight.FlightNumber},{booking.User.Id},{booking.BookingDate},{booking.Status}";
+    }
+
+    public static string header = "bookingId,flightNumber,userId,bookingDate,status";
+
+    public Booking(int id, User user, Flight flight)
+    {
+        Id = id;
+        User = user;
+        Flight = flight;
+        Status = BookingStatus.Confirmed;
+    }
+
+    public static Booking FromCsv(string csv)
+    {
+        string[] values = csv.Split(',');
+        Flight? flight = Flight.GetFlightByNumber(int.Parse(values[1]));
```

Comment:
```suggestion
        var values = csv.Split(',');
        var flight = Flight.GetFlightByNumber(int.Parse(values[1]));
```

When it's easy to predict the variable type, then it's preferable to use `var` instead of the explicit type. You can check this article: https://intellitect.com/blog/when-to-use-and-not-use-var-in-c/